src/card.tsx

```diff
@@ -726,3 +732,199 @@ export const DisplayDataCard = React.forwardRef<HTMLDivElement, DisplayDataCardP
         />
     )
 );
+
+interface PosterCardBaseProps {
+    backgroundImage: string;
+    ariaLabel?: string;
+    aspectRatio?: AspectRatio | number;
+    width?: number | string;
+    height?: number | string;
+    icon?: React.ReactElement;
+    actions?: Array<CardAction>;
+    onClose?: () => void;
+    dataAttributes?: DataAttributes;
+
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    title?: string;
+    titleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+}
+
+interface PosterCardToProps extends PosterCardBaseProps {
+    to?: string;
+    fullPageOnWebView?: boolean;
+}
+
+interface PosterCardHrefProps extends PosterCardBaseProps {
+    href?: string;
+    newTab?: boolean;
+}
+
+interface PosterCardOnPressProps extends PosterCardBaseProps {
+    onPress?: () => void;
+}
+
+type PosterCardProps = ExclusifyUnion<PosterCardToProps | PosterCardHrefProps | PosterCardOnPressProps>;
```

Comment:
the typing gave me some headache, check it carefully. And even more important: if you see a better way to handle this, do tell!
For instance: I've added the function below to make things work properly `getPosterTouchableProps`